app/hotels/package.json

```diff
@@ -7,11 +7,14 @@
     "@kiwicom/react-native-app-config": "^0",
     "@kiwicom/react-native-app-redux": "^0",
     "@kiwicom/react-native-app-shared": "^0",
+    "enzyme": "^3.3.0",
+    "enzyme-adapter-react-16": "^1.1.1",
```

Comment:
Should be part of the root package JSON if you decided to use Enzyme for this project.

Reply:
Ok, I will create an issue to move it to root package